Filter/ModelFilter.php

```diff
@@ -100,9 +100,11 @@ protected function handleMultiple(ProxyQueryInterface $queryBuilder, $alias, $da
 
             $or->add($queryBuilder->expr()->notIn($alias, ':'.$parameterName));
 
-            $or->add($queryBuilder->expr()->isNull(
-                sprintf('IDENTITY(%s.%s)', $this->getParentAlias($queryBuilder, $alias), $this->getFieldName())
-            ));
+            if (ClassMetadataInfo::MANY_TO_MANY !== $this->getOption('mapping_type')) {
```

Comment:
You can't assign anything to `$this->getOption('mapping_type')` since it isn't a variable, so the Yoda conditions is unnecessary. Should Yoda be used anyway @sonata-project/contributors ?